fs/fs_zenfs.cc

```diff
@@ -1254,13 +1255,14 @@ Status ZenFS::Mount(bool readonly) {
   return Status::OK();
 }
 
-Status ZenFS::MkFS(std::string aux_fs_path, uint32_t finish_threshold) {
+Status ZenFS::MkFS(std::filesystem::path const& aux_fs_path,
```

Comment:
The line `Superblock super(zbd_, aux_fs_path, finish_threshold);` should be converted to `Superblock super(zbd_, aux_fs_path.string(), finish_threshold);`

Reply:
I think passing the path and using implicit conversion is fine. There is a conversion function on `path` to `string_type`.
 
* [conversion operator](https://en.cppreference.com/w/cpp/language/cast_operator)
* [implicit conversion](https://en.cppreference.com/w/cpp/language/implicit_conversion)
* [conversion operator on path](https://en.cppreference.com/w/cpp/filesystem/path/native)